apps/lcs/sections/summary-data-sections.js

```diff
@@ -14,7 +14,52 @@ module.exports = {
       {
         step: '/property-occupied',
         field: 'when-person-moved-in',
+        parse: (value, req) => {
+          if (!req.sessionModel.get('steps').includes('/property-occupied')) {
+            return null;
+          }
+          if(req.sessionModel.get('person-live-in') === 'no') {
+            return 'Not Applicable';
+          }
```

Comment:
I would suggest to return `null` instead, to hide this step if not applicable
Could you please confirm with UCD team